graphene_sqlalchemy/tests/test_types.py

```diff
@@ -181,3 +181,36 @@ class Meta:
         resolver, TestConnection, ReporterWithCustomOptions, None, None
     )
     assert result is not None
+
+
+class HumanWithFieldAliased(SQLAlchemyObjectType):
+
+    publication_timestamp = Int()
+
+    class Meta:
+        model = Article
+        registry = registry
+        interfaces = (Node,)
+        aliased_fields = {
+            "id": "article_id",
+            "headline": "title",
+            "pub_date": "publication_timestamp",
```

Comment:
Maybe don't rename *all* keys but leave one original name to test if that works, too.

Reply:
Maybe also add a similar test to `test_reflected.py` because that is the case where this is most useful in practice.